src/engine/renderer/tr_local.h

```diff
@@ -292,6 +292,9 @@ static inline void glFboSetExt()
 	GL_fboShim.glRenderbufferStorageMultisample = glRenderbufferStorageMultisampleEXT; */
 }
 
+enum class Light { FullBright, Vertex, Grid, Map };
```

Comment:
The naming on these is pretty far from our conventions. I guess I don't insist on '_t' but something like lighting style would be more descriptive than just "light". The values (e.g. FullBright, Vertex) should be all caps.

Reply:
Light Style refers to something else in engine, light styles are extra lightmaps dynamically blended in `.shader`.

Light Mode would be better.

Reply:
I renamed them `lightMode_t` and `deluxeMode_t` and set values all caps.